### PR TITLE
Gallery: use fstrings for better readability

### PR DESCRIPTION
**Pull request by sciunto**

### CODE REVIEW

File: doc/examples/transform/plot_masked_register_translation.py [hunk 1]
```diff
@@ -33,23 +33,23 @@
 image = data.camera()
 shift = (-22, 13)
 
-corrupted_pixels = np.random.choice([False, True], 
-                                    size = image.shape, 
+corrupted_pixels = np.random.choice([False, True],
+                                    size = image.shape,
```
Comment (soupault) [inline doc/examples/transform/plot_masked_register_translation.py]: This looks bad 😄. Also, the same issue below. I'll fix that once I review that PR history.

**Comment by hmaarrfk** [comment]
Whats the review process here? Are we expected to look at the resulting doc build? These kind of changes are cosmetic, and often end up trading one set of bugs for an other.

I would be a little less eager to add f-strings to existing code, but since you've done it already, this is fine.
The existing code was fine, and in many cases, only had one or two variables printed out per line, so the curly brace formatting was fine OK.

**Comment by sciunto** [comment]
I did it for the gallery as this is the code people tend to replicate. From my experience, fstring is much easier to explain to students learning python than any other option.

I agree to do not generalize the idea to the existing code of the library.

**Comment by hmaarrfk** [comment]
> fstring is much easier to explain to students learning python

I've found it pretty hard to explain to people transitioning from py 2

**Comment by JDWarner** [comment]
Agree these are harder to explain at a whiteboard, but really easy to get people to catch on to by showing examples and experimenting (actually, this PR's diff would be a decent teaching tool).  The readability is a lot better.

**Pull request merged**